src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -767,7 +767,9 @@ uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
         } else {
             md->super.config.odp.max_size = 1ul << 28;
         }
+    }
 
+    if (UCT_IB_MLX5DV_GET(cmd_hca_cap, cap, umr_extended_translation_offset)) {
```

Comment:
this needs to be a part of another pr in UCT

Reply:
Yes, I just add it here in testing purpose. BTW I think that it won't bring any harm if we merge before this UCT PR with this change.

Reply:
yes, but we need to remove it from this PR